### DISCUSSION TITLE
[Question] Android Build problems

### DISCUSSION THREAD

**Discussion by MoreiraCristiano** [flet-dev/flet#2644] [Q&A] [answered]
### Question

I am building my app for android, but after create the apk and install I am having the issue 
- TypeError: RegisterWebClientRequestPayload._init_() missing 1 required positional argument: 'media'

Do you guys already had a similar problem?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment, marked as answer]
The new Flet 0.20.0 came yesterday and it has some changes in the API.

Two ways to fix the issue you have:

1) Upgrade to the latest Flet 0.20.0 and build your app again:

```
pip install flet --upgrade
```

or, if you want to stay on Flet 0.19.0:

2) Pin `flet` dependency in `requirements.txt` to version 0.19.0:


```
flet==0.19.0
```
